shared/js/initApp.js

```diff
@@ -128,7 +138,7 @@ dashboard.saveProject = function(callback, overrideSource) {
   dashboard.currentApp.levelHtml = window.Applab && Applab.getHtml();
   dashboard.currentApp.level = appToProjectUrl[appOptions.app];
   if (channelId && dashboard.currentApp.isOwner) {
-    channels().update(channelId, dashboard.currentApp, function(data) {
+    channels().update(channelId, dashboard.currentApp, function(callback, data) {
```

Comment:
it appears to me that a second call was made to saveProject and channels().update before the first call to channels().update returned, that this callback would have the value passed in by the second call rather than the value passed in by the first call.